healthyregions/chicago-environment-explorer
# feat: added 1995 Heat Wave Deaths as a new variable

## Problem
We have a new variable submission to ChiVes: 1995 Heath Wave Deaths

Fixes #147 

## Approach
* feat: added the 1995 Heat Wave Deaths as a new variable under the "Health" category
* fix: rename / reorder new overlay layers
    * Rename "Industrial and Non-Residential Areas" layer to "Industrial & Non-Residential Zoning"
    * Rename this new layer to "Airports, Railways, & Major Roadways"
    * Reorder "Airports, Railways, & Major Roadways" layer beneath the "Industrial & Non-Residential Zoning" layer in the drop down

![Screenshot 2024-08-19 at 4 43 51 PM](https://github.com/user-attachments/assets/4c9f1907-e496-4e14-9857-004f5b2dcbbd)

## How to Test
1. Navigate to https://deploy-preview-199--chicago-env-explorer.netlify.app/map
2. From the Variables dropdown, scroll down to "Health" and choose "1995 Heat Wave Deaths"
    * You should see the new variable data visualized on the map
